### PR TITLE
Attempt to quiet @chesster in the scheduling channels.

### PR DESCRIPTION
**Pull request by lakinwecker**

### CODE REVIEW

File: spreadsheets.js [hunk 1]
```diff
@@ -9,17 +9,94 @@ var EXTREMA_DEFAULTS = {
...
+    "HH:mm",
+    "H:mm",
+    "H:m"
+];
 var DATE_FORMATS = [
-    "YYYYMMDD HHmm",
-    "HHmm DDMMYYYY"
 ];
+BASE_DATE_FORMATS.forEach(function(date_format) {
+    BASE_TIME_FORMATS.forEach(function(time_format) {
```
Comment (endrawes0) [inline spreadsheets.js]: Nice use of static data.

File: spreadsheets.js [hunk 2]
```diff
@@ -9,17 +9,94 @@ var EXTREMA_DEFAULTS = {
...
+        var time_format2 = time_format.replace(/[:]/g, '');
+        DATE_FORMATS.push(date_format + " " + time_format);
+        DATE_FORMATS.push(time_format + " " + date_format);
+        DATE_FORMATS.push(date_format2 + " " + time_format2);
+        DATE_FORMATS.push(time_format2 + " " + date_format2);
+        DATE_FORMATS.push(date_format2 + " " + time_format);
+        DATE_FORMATS.push(time_format + " " + date_format2);
+        DATE_FORMATS.push(date_format + " " + time_format2);
+        DATE_FORMATS.push(time_format2 + " " + date_format);
+    });
```
Comment (endrawes0) [inline spreadsheets.js]: This structure just got much bigger

Comment (lakinwecker) [inline spreadsheets.js]: Yeah, it's huge and slow.  Which is why I need to short circuit the date parsing below.

Comment (endrawes0) [inline spreadsheets.js]: Yes, i saw that. Not much you can do though. Unless we can order these in some way, brute force is out only option.

**Comment by endrawes0** [comment]
Looks fine. Pulling.

**Pull request merged**